api/apps/geoprocessing/src/marxan-sandboxed-runner/adapters-blm/blm-partial-results.geo.entity.ts

```diff
@@ -35,16 +35,16 @@ export class BlmPartialResultEntity {
   blmValue!: number;
 
   /**
-   * Score of the run
+   * Cost output parameter
    */
   @Column({
-    name: `score`,
+    name: `cost`,
     type: 'float',
```

Comment:
REST Api results should rather be DB-agnostic. If needed, such mapping should be done on controller level (serializer) or could even be done at `sql` level using `as`

Reply:
not clear what in here is postgres specific.